app/src/renderer/vuex/modules/send.js

```diff
@@ -14,6 +14,12 @@ export default ({ node }) => {
   }
 
   async function doSend({ state, dispatch, commit, rootState }, args) {
+    if (!rootState.connection.connected) {
+      throw Error(
+        `Currently not connected to a secure node. Please try again when Voyager has secured a connection.`
```

Comment:
what are the steps to secure a connection ? if it's just waiting we should display  `Please wait until Voyager has secured a connection`

Reply:
This should not happen. All interaction should be disabled if not connected. This is just in case we screwed up. And in that case we currently don't send a transaction again automatically.